From 2b8cf62e2812210402ba164e42e65065d17e7f50 Mon Sep 17 00:00:00 2001
From: Julien Massot <jmassot@softbankrobotics.com>
Date: Thu, 20 Jul 2017 09:11:32 +0200
Subject: [PATCH 1/5] dbus: Rename D-Bus autostart service file name

The autostart file needs to match the service name according the D-Bus
specification [1]:

"""
The service filename of "org.me.test.service" is then searched for
in /usr/share/dbus-1/system-services or other specified directories.
"""

This fixes "Bus name not found in system service directory".

[1] https://dbus.freedesktop.org/doc/system-activation.txt
---
 Makefile.am                                            | 2 +-
 configure.ac                                           | 2 +-
 src/{pacrunner.service.in => org.pacrunner.service.in} | 0
 3 files changed, 2 insertions(+), 2 deletions(-)
 rename src/{pacrunner.service.in => org.pacrunner.service.in} (100%)

diff --git a/Makefile.am b/Makefile.am
index 414a3e3..4cd0971 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -8,7 +8,7 @@ dbusconf_DATA = src/pacrunner.conf
 
 dbusdatadir = @DBUS_DATADIR@
 
-dbusdata_DATA = src/pacrunner.service
+dbusdata_DATA = src/org.pacrunner.service
 endif
 
 gdbus_sources = gdbus/gdbus.h gdbus/mainloop.c gdbus/watch.c \
diff --git a/configure.ac b/configure.ac
index c68adae..1577c86 100644
--- a/configure.ac
+++ b/configure.ac
@@ -150,4 +150,4 @@ AC_ARG_ENABLE(datafiles, AC_HELP_STRING([--disable-datafiles],
 					[enable_datafiles=${enableval}])
 AM_CONDITIONAL(DATAFILES, test "${enable_datafiles}" != "no")
 
-AC_OUTPUT(Makefile src/pacrunner.service libproxy/libproxy-1.0.pc)
+AC_OUTPUT(Makefile src/org.pacrunner.service libproxy/libproxy-1.0.pc)
diff --git a/src/pacrunner.service.in b/src/org.pacrunner.service.in
similarity index 100%
rename from src/pacrunner.service.in
rename to src/org.pacrunner.service.in
-- 
2.13.3

